### DISCUSSION TITLE
BUILD ERROR WITH JENKLL

### DISCUSSION THREAD

**Discussion by stvnpalma** [community/community#132805] [Repositories]
### Select Topic Area

General

### Body

Hi all,

I am having the same issue here. Could someone assist me, becuase if I change from /docs to /root I don't get the error the actual site deploys fine and creates a pages URL perfectly, nevertheless when I open it doesn't render the content.

build Logging at level: debug GitHub Pages: github-pages v231 GitHub Pages: jekyll v3.9.5 Theme: jekyll-theme-primer Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0 Requiring: jekyll-github-metadata Requiring: jekyll-seo-tag Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings GitHub Metadata: Initializing... Source: /github/workspace/./docs Destination: /github/workspace/./docs/_site Incremental build: disabled. Enable with --incremental Generating... Generating: JekyllOptionalFrontMatter::Generator finished in 8.005e-06 seconds. Generating: JekyllReadmeIndex::Generator finished in 4.008e-06 seconds. Generating: Jekyll::Paginate::Pagination finished in 2.254e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 2.0749e-05 seconds. Generating: JekyllDefaultLayout::Generator finished in 9.227e-06 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 5.751e-06 seconds. Rendering: assets/css/style.scss Pre-Render Hooks: assets/css/style.scss Rendering Markup: assets/css/style.scss github-pages 231 | Error: No such file or directory @ dir_chdir - /github/workspace/docs
[build](https://github.com/stvnplm8/Portfolio/actions/runs/10010103906/job/27670529707)
Logging at level: debug GitHub Pages: github-pages v231 GitHub Pages: jekyll v3.9.5 Theme: jekyll-theme-primer Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0 Requiring: jekyll-github-metadata Requiring: jekyll-seo-tag Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings GitHub Metadata: Initializing... Source: /github/workspace/./docs Destination: /github/workspace/./docs/_site Incremental build: disabled. Enable with --incremental Generating... Generating: JekyllOptionalFrontMatter::Generator finished in 8.005e-06 seconds. Generating: JekyllReadmeIndex::Generator finished in 4.008e-06 seconds. Generating: Jekyll::Paginate::Pagination finished in 2.254e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 2.0749e-05 seconds. Generating: JekyllDefaultLayout::Generator finished in 9.227e-06 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 5.751e-06 seconds. Rendering: assets/css/style.scss Pre-Render Hooks: assets/css/style.scss Rendering Markup: assets/css/style.scss github-pages 231 | Error: No such file or directory @ dir_chdir - /github/workspace/docs

**Comment by knarrff** [comment]
Hi @stvnplm8 ,

it looks like you do not even want to build using Jekyll, is this correct? If so, you can disable that by creating a file in your repository with the name ```.nojekyll``` (see [docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators)).

The reason you do get this now and not earlier is also mentioned there: "_If you publish your site from a source branch, GitHub Pages will use Jekyll to build your site by default._"

**Reply by stvnpalma** [reply]
Hi @knarrff thank you for reaching out, on. the other hand if I push it from my /root when I click on on the pages URL created it won't display the content. Any ideas? Or what could I be doing wrong?

**Reply by knarrff** [reply]
You page does have content. Take a look at view-source:https://stvnplm8.github.io/Portfolio/ . This is what is in https://github.com/stvnplm8/Portfolio/blob/main/index.html . The reason why the page https://stvnplm8.github.io/Portfolio/ looks empty is that, well, your html body does not contain anything that is displayed, only a javascript module, but that module isn't actually javascript, but html, and thus, isn't loaded: "Loading module from “https://stvnplm8.github.io/src/main.tsx” was blocked because of a disallowed MIME type (“text/html”)."

**Reply by knarrff** [reply]
I just noticed: your html code says ```<script type="module" src="/src/main.tsx"></script>```, which is turned into https://stvnplm8.github.io/src/main.tsx but the file is actually served here: https://stvnplm8.github.io/Portfolio/src/main.tsx (note the extra "Portfolio") This is also why the mime-type does not match: you get an html-error-page. You should either use a relative path (src="src/main.tsx") or include this part into the absolute path as well (src=/Portfolio/src/main.tsx").

**Reply by stvnpalma** [reply]
@knarrff for some reason I tried both your suggestions yet still getting a white page when I launch it.

**Reply by knarrff** [reply]
As far as I can see from this https://github.com/stvnplm8/Portfolio/commit/df9bf2c361cf73ead79f2f67899e806949d9d5b9 and similar, you changed it to

- ```src="src=/Portfolio/src/main.tsx"```
- ```src="src=/main.tsx"```

and now back to

- ```src="/src/main.tsx"```

note the extra ```"src=``` in the first two: an easy to miss copy&paste mistake.

**Reply by stvnpalma** [reply]
@knarrff Thank you. I did all no luck and had to switch it back so it would run on my machine again.

**Comment by ShahinIsmayilov** [comment]
Response #1

**Comment by ShahinIsmayilov** [comment]
Response #2
